gpu-service/Dockerfile

```
@@ -29,8 +30,7 @@ ENV PATH="/opt/venv/bin:$PATH"
 # Copy and install Python dependencies into the virtual environment
 COPY requirements.txt /app/requirements.txt
 WORKDIR /app
-RUN pip install --no-cache-dir -r requirements.txt && \
-    pip install --no-cache-dir git+https://github.com/ageitgey/face_recognition_models
+RUN pip install -r requirements.txt
```

Hadolint (2.12.0)

[warning] Avoid use of cache directory with pip. Use `pip install --no-cache-dir <package>`

(DL3042)